bin/runner.go

```diff
@@ -22,11 +22,16 @@ func main() {
 	clients := utils.ClientSets{}
 	// Getting kubeConfig and Generate ClientSets
 	if err := clients.GenerateClientSetFromKubeConfig(); err != nil {
-		log.Fatalf("Unable to create ClientSets, error: %v", err)
+		log.Errorf("Unable to create ClientSets, error: %v", err)
```

Comment:
Please don't capitalise initial letters of log message

Reply:
updated